pkg/api/types.go

```diff
@@ -1753,6 +1753,32 @@ type ReplicationControllerStatus struct {
 
 	// ObservedGeneration is the most recent generation observed by the controller.
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+
```

Comment:
This API change LGTM - it is consistent with other objects that have conditions.

@kubernetes/api-review-team any disagreement?